batch/sql/add-seqr-nonpreemptible-pools.sql

```diff
@@ -0,0 +1,44 @@
+-- Adds dedicated pools for the seqr loading pipeline, with the 'seqr' pool label, but non-preemptible versions 
+-- run after add-seqr-pools.sql and add-seqr-nonpreemptible-pools.sql
```

Comment:
Did you mean `add-nonpreemptible-pools.sql`?

```suggestion
-- run after add-seqr-pools.sql and add-nonpreemptible-pools.sql
```